the-devastating-chickens/BangazonAPI
# Sg bangazon api

# Description

Implementation of all CRUD actions for product type
Issue ticket #3.

## Type of change

- [ ] User can be able to GET a list, and GET a single item, PUT, POST and DELETE Product type and a test for it. 

# Testing Instructions

- Git Fetch --all
- Git Checkout sg-BangazonAPI
- Run the API and check if it does GET, POST, PUT, DELETE product type resource
- Run the test as well and see if it pass all the testes.  

# Checklist:
